run-once.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
The `run-once.sh` and `run_instrumented.sh` scripts
* they don't belong in the root path.
* have system specific information (i.e. Mare Nostrum).
* the naming, with underscore and hyphen, is inconsistent.

I would make a new path `scripts/system-marenostrum` and put them in their (along with `extrae.xml`)